src/main.rs

```diff
@@ -77,7 +77,38 @@ use std::process::Command;
 use regex::Regex;
 use std::fs;
 
+// absolutely awful hack to fix the sysroot when running out-of-tree
+// Taken from #78926, which took it from src/bin/miri.rs, which in turn took it from clippy ... rustc is a mess.
+// FIXME(jyn514): implement https://github.com/rust-lang/rust/pull/78926#issuecomment-726653035 instead
```

Comment:
Everything about this is awful, but not much I can do better until fixing the issue I linked.